README.md

```diff
@@ -173,7 +173,7 @@ There are five test suites in the `e2e/interchaintestv8` directory:
 Where `$TEST_NAME` is the name of the test you want to run, for example:
 
 ```sh
-just test-e2e TestDeploy_Groth16
+just test-e2e-eureka TestDeploy_Groth16
 ```
```

Comment:
Is this actually needed though

Reply:
Yeah, if you run the previous version of this, it doesn't find any tests to run.

Reply:
I meant the whole block. It seems out of place in README

Reply:
Sure, I guess we could remove it. @mconcat can you remove this block, since we have concrete examples of how to run tests from different suites there already?